recipes/pygridgen/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+    name: pygridgen
+    version: 0.1.1
+
+source:
+    git_url: https://github.com/phobson/pygridgen.git
+    git_tag: 0.1.1
+
+build:
+    number: 0
+    skip: True  # [win]
+    script: python setup.py install
```

Comment:
Totally fine. I have had a few cases like that too. Just wanted to confirm. Thanks.